tensorboard/backend/event_processing/plugin_event_accumulator_test.py

```diff
@@ -116,30 +117,30 @@ class MockingEventAccumulatorTest(EventAccumulatorTest):
   def setUp(self):
     super(MockingEventAccumulatorTest, self).setUp()
     self.stubs = tf.test.StubOutForTesting()
-    self._real_constructor = ea.EventAccumulator
-    self._real_generator = ea._GeneratorFromPath
+    self._real_constructor = ea.EventAccumulatorPlugin
```

Comment:
If the class isn't renamed, the churn in this file should hopefully not be necessary. Churn makes it more difficult to `git blame`.